images-list

```diff
@@ -223,6 +223,7 @@ gcr.io/kubebuilder/kube-rbac-proxy rancher/kube-rbac-proxy v0.5.0
 gcr.io/kubebuilder/kube-rbac-proxy rancher/mirrored-kube-rbac-proxy v0.5.0
 gcr.io/kubebuilder/kube-rbac-proxy rancher/mirrored-kube-rbac-proxy v0.14.0
 gcr.io/kubebuilder/kube-rbac-proxy rancher/mirrored-kube-rbac-proxy v0.15.0
+gcr.io/kubebuilder/kube-rbac-proxy rancher/mirrored-kube-rbac-proxy v0.18.0
```

Comment:
I don't see anything in upstream newer than v0.16.0. Is the upstream project now publishing somewhere else?

Reply:
yeah I've been looking into that. Looks like the upstream chart changed their upstream to `brancz/kube-rbac-proxy` which wasn't clear when I was rebasing the chart. Will need a new repo

Reply:
Looking at the repo, I suspect that the gcr.io/kubebuilder/kube-rbac-proxy image was always just a mirror of the `brancz/kube-rbac-proxy` image since the project still hasn't been accepted into the kubernetes org. Kind of an odd situation.